.github/workflows/upload-sbom.yaml

```diff
@@ -0,0 +1,26 @@
+# .github/workflows/sbom.yaml
```

Comment:
Well that was supposed to be an indication to rename the file, not add this line. Sorry 🙂

Reply:
👍